src/humancellatlas/data/metadata/api.py

```diff
@@ -279,6 +279,13 @@ def contact_name(self) -> str:
                       f"Use ProjectContact.name instead.", DeprecationWarning)
         return self.name
 
+@dataclass(init=False)
+class Accession:
+
```

Comment:
```suggestion
class Accession:
    type: str
    value: str

```

Consider:
```
# without member variables defined in @dataclass

>>> from humancellatlas.data.metadata.api import Accession
>>> a = Accession('foo', 'bar')
a
Accession()

# with member variables defined in @dataclass

>>> from humancellatlas.data.metadata.api import Accession
>>> a = Accession('foo', 'bar')
a
Accession(type='foo', value='bar')
```

Reply:
Ah! that's why in the debugger it was not showing any information about the `type,value` when inspecting project.accessions, thanks! :)